src/main/java/org/openpnp/gui/LogPanel.java

```diff
@@ -157,7 +158,16 @@ public void contentsChanged(ListDataEvent e) {
 
     private void scrollDownLogPanel() {
         if (autoScroll) {
-            logEntryJList.ensureIndexIsVisible(logEntries.getSize() - 1);
+            JScrollBar verticalBar = scrollPane.getVerticalScrollBar();
+            AdjustmentListener downScroller = new AdjustmentListener() {
+                @Override
+                public void adjustmentValueChanged(AdjustmentEvent e) {
+                    Adjustable adjustable = e.getAdjustable();
+                    adjustable.setValue(adjustable.getMaximum());
+                    verticalBar.removeAdjustmentListener(this);
+                }
+            };
+            verticalBar.addAdjustmentListener(downScroller);
```

Comment:
That is true, the problem if you keep the Listener attached is that it does not allow scrolling up if autoscroll is enabled. I will check if i can improve this behaviour during the weekend, maybe it is possible to distinguish user events and data added events